chain/pool/txpool.go

```diff
@@ -487,6 +572,7 @@ func (tp *TxnPool) verifyTransactionWithLedger(txn *transaction.Transaction) err
 		if len(id) != 0 {
 			return errors.New("ID has be registered")
 		}
+	case pb.NanoPayType:
```

Comment:
Is there some checks with ledger?

Reply:
I don't think so, I think using state is enough (which we should try to achieve for all txn types). But I will wait @trueinsider to confirm.

Reply:
Actually, I'm confused. In theory there should be exactly the same validation as in txValidator.go. In master they even use exactly the same functions. I'm not sure why it become separate again when we switched to balance.

Reply:
I agree. I think we should do that clean soon

Reply:
done in separate PR